src/js/views/patients/shared/components/check_component.js

```diff
@@ -0,0 +1,33 @@
+import hbs from 'handlebars-inline-precompile';
+import { Component } from 'marionette.toolkit';
+
+import 'sass/modules/buttons.scss';
+
+export default Component.extend({
+  stateEvents: {
+    'change:isSelected': 'onStateChangeIsSelected',
+  },
+  viewEvents: {
+    'click': 'onClick',
+  },
+  viewOptions() {
+    const isSelected = this.getState('isSelected');
+    const template = isSelected ? hbs`{{fas "check-square"}}` : hbs`{{fal "square"}}`;
+    return {
```

Comment:
_Nitpick alert_.

Add space between  the `const template` and the `return {` lines?